server/controllers/notifications.js

```diff
@@ -0,0 +1,99 @@
+const mongoose = require("mongoose")
+const asyncHandler = require("express-async-handler")
+const decodeToken = require("../utils/decodeToken")
+const jwt = require("jsonwebtoken")
+
+const Notification = require("../models/Notifications")
+
+// @route GET /notifications/all
+// Get all notifications
+exports.getAllNotifications = asyncHandler(async (req, res) => {
+  let decoded = decodeToken(req.cookies.token)
```

Comment:
We don't need to decode the token from cookies here, it has already been done by the auth middleware. You can access the user's id from the request object - `req.user.id`